### PR TITLE
Doc COTS: precisions on a new trip

### PR DESCRIPTION
**Pull request by papailio**

### CODE REVIEW

File: documentation/cots_connector.md [hunk 1]
```diff
@@ -77,6 +77,16 @@ In case of a trip addition, when the *nouvelleVersion/indicateurFer* is set to `
 
 The operator associated to the considered Navitia trips should have a complementary code of type `RefProd` that matches the value of *nouvelleVersion/codeCompagnieTransporteur*. If the field is empty in the COTS stream, then the value `1187` that corresponds to the SNCF operator is used.
 
+#### Creating a new VehicleJourney for Navitia
+In case of a trip addition (trip status is set to `add`), a complete `VehicleJourney` needs to be sent to Navitia using the available information in the COTS stream.
+* The new trip is associated with the SNCF operator.
+* The trip headsign is derived from the train number in *nouvelleVersion/numeroCourse*.
```
Comment (pbougue) [inline documentation/cots_connector.md]: What do you mean by "derived"?
The current plan is to use it directly.

File: documentation/cots_connector.md [hunk 2]
```diff
@@ -77,6 +77,16 @@ In case of a trip addition, when the *nouvelleVersion/indicateurFer* is set to `
...
+* The physical mode is set to `Coach` when the value of *nouvelleVersion/indicateurFer* is `ROUTIER`, otherwise the physical mode is set to `LongDistanceTrain`.
+* A new dataset as well as a new service attached to the new trip must be created in Navitia.
```
Comment (pbougue) [inline documentation/cots_connector.md]: We discussed today in refinement to create a dedicated dataset in navitia (probably named `realtime.cots` like what's done on "contributor" field of disruptions).
This dataset being attached to SNCF's contributor (from NTFS).

File: documentation/cots_connector.md [hunk 3]
```diff
@@ -77,6 +77,16 @@ In case of a trip addition, when the *nouvelleVersion/indicateurFer* is set to `
...
+* The new trip is attached to a new route named after the origin and destination stops.
+* The new route is attached to a new line.
```
Comment (pbougue) [inline documentation/cots_connector.md]: The line will have the same name as the route (O-D).

File: documentation/cots_connector.md [hunk 4]
```diff
@@ -77,6 +77,16 @@ In case of a trip addition, when the *nouvelleVersion/indicateurFer* is set to `
 
 The operator associated to the considered Navitia trips should have a complementary code of type `RefProd` that matches the value of *nouvelleVersion/codeCompagnieTransporteur*. If the field is empty in the COTS stream, then the value `1187` that corresponds to the SNCF operator is used.
 
+#### Creating a new VehicleJourney for Navitia
+In case of a trip addition (trip status is set to `add`), a complete `VehicleJourney` needs to be sent to Navitia using the available information in the COTS stream.
+* The new trip is associated with the SNCF operator.
```
Comment (pbougue) [inline documentation/cots_connector.md]: "operator" notion is vague to me, are you refereing to navitia's company?

File: documentation/cots_connector.md [hunk 5]
```diff
@@ -77,6 +77,16 @@ In case of a trip addition, when the *nouvelleVersion/indicateurFer* is set to `
...
+* The trip headsign is derived from the train number in *nouvelleVersion/numeroCourse*.
+* The physical mode is set to `Coach` when the value of *nouvelleVersion/indicateurFer* is `ROUTIER`, otherwise the physical mode is set to `LongDistanceTrain`.
+* A new dataset as well as a new service attached to the new trip must be created in Navitia.
+* The new trip is attached to a new route named after the origin and destination stops.
+* The new route is attached to a new line.
+* The commercial mode of the new line is determined based on the value of *nouvelleVersion/codeMarqueTransporteur*. When the value is `TER`, if the *nouvelleVersion/indicateurFer* is `FERRE`, then the commercial mode is set to `Train TER`. In case of a new road trip, the commercial mode is set to `Car TER`. In all other cases, the commercial mode is set after the value of *nouvelleVersion/codeMarqueTransporteur*.
```
Comment (pbougue) [inline documentation/cots_connector.md]: This is too vague (but maybe not your role to answer it):
* should we create the commercial_modes if they are absent?
* "Train TER" and "Car TER" means we should search them by their (non-unique) names?
* This just covers one value of "codeMarqueTransporteur", what about the others, and how can we automatize that?

File: documentation/cots_connector.md (hunk 4 above)
Comment (mbounabi) [reply documentation/cots_connector.md]: SNCF I guess. the "SNCF operator"

Comment (pbougue) [reply documentation/cots_connector.md]: As described in the TripUpdate section: company_id == operator, so we let it as-is.

File: documentation/cots_connector.md (hunk 5 above)
Comment (pbougue) [reply documentation/cots_connector.md]: Decided to just list it as an idea.

File: documentation/cots_connector.md [hunk 6]
```diff
@@ -77,6 +77,16 @@ In case of a trip addition, when the *nouvelleVersion/indicateurFer* is set to `
 
 The operator associated to the considered Navitia trips should have a complementary code of type `RefProd` that matches the value of *nouvelleVersion/codeCompagnieTransporteur*. If the field is empty in the COTS stream, then the value `1187` that corresponds to the SNCF operator is used.
 
+#### Creating a new VehicleJourney for Navitia
+In case of a trip addition (trip status is set to `add`), a complete `VehicleJourney` needs to be sent to Navitia using the available information in the COTS stream.
+* The new trip is associated with the SNCF operator.
+* The trip headsign is derived from the train number in *nouvelleVersion/numeroCourse*.
+* The physical mode is set to `Coach` when the value of *nouvelleVersion/indicateurFer* is `ROUTIER`, otherwise the physical mode is set to `LongDistanceTrain`.
```
[the remark this answers is not in the document]
Comment (kadhikari) [reply documentation/cots_connector.md]: Ok for the current version

**Pull request merged**